pump/server.go

```diff
@@ -597,6 +600,23 @@ func (s *Server) BinlogByTS(w http.ResponseWriter, r *http.Request) {
 		fmt.Fprint(w, "\n\n PrewriteValue: \n")
 		fmt.Fprint(w, prewriteValue.String())
 	}
+
+	if len(binlog.PrewriteKey) > 0 {
+		tikvStorage := s.tiStore.(tikv.Storage)
+		healper := storage.Helper{
```

Comment:
why not one `Helper` instance?

Reply:
no performance cost and only use here so I think better just construct here, don't make Server struct complecate